packages/react-strict-dom/src/native/modules/createStrictDOMComponent.js

```diff
@@ -142,12 +142,18 @@ export function createStrictDOMComponent<T, P: StrictProps>(
         nativeProps.experimental_layoutConformance = 'strict';
       }
 
-      // $FlowFixMe
-      let element = <NativeComponent {...nativeProps} />;
+      let element: React.Node =
+        typeof props.children === 'function' ? (
```

Comment:
So technically, any component accepts function as children now and we're using types to ensure it isn't abused this way?

Reply:
That's right, but only the native components accept function children. Maybe I can add another arg to the factory functions to prevent runtime use without types

Reply:
I think this is fine, was just making sure I understood the implementation.